src/openrct2/actions/MazeSetTrackAction.hpp

```diff
@@ -69,10 +69,8 @@ DEFINE_GAME_ACTION(MazeSetTrackAction, GAME_COMMAND_SET_MAZE_TRACK, GameActionRe
 
     void Serialise(DataSerialiser & stream) override
     {
-        GameAction::Serialise(stream);
-        uint16_t _x = _loc.x , _y = _loc.y , _z = _loc.z;
-        Direction _direction = _loc.direction;   
-        stream << DS_TAG(_x) << DS_TAG(_y) << DS_TAG(_z) << DS_TAG(_direction) << DS_TAG(_initialPlacement) << DS_TAG(_rideIndex) << DS_TAG(_mode);
+        GameAction::Serialise(stream);   
+        stream << DS_TAG(_loc.x) << DS_TAG(_loc.y) << DS_TAG(_loc.z) << DS_TAG(_loc.direction) << DS_TAG(_initialPlacement) << DS_TAG(_rideIndex) << DS_TAG(_mode);
```

Comment:
You can just pass loc as an individual item there should already be a serialisation for it